### PR TITLE
Add reset sandbox documentation

### PR DESCRIPTION
**Pull request by wandi34**
# Summary

Add reset sandbox data documentation

# Checklist for Definition of Done

Depending on the task some of the following jobs are not relevant. Please check the items or cross them out (`~~`text`~~`).

* [x] Code is technically reviewed by myself
* [x] Code is manually tested
* [x] Pipeline is green
* [x] Automatic tests for new functionality are created
* [x] Documentation is updated
* [x] Branch is rebased and therefore ready to merge
* [x] Git history is clean

### CODE REVIEW

File: arc42/src/07_deployment_view.adoc [hunk 1]
```diff
@@ -248,6 +248,13 @@ The CLI can echo the full schema instead of doing the actual migration, too. Thi
 $ java -jar sandbox-{acc-version}.jar create-schema --spring.datasource.url=$JDBC_URL --spring.datasource.username=$USR --spring.datasource.password=$PASS
 ----
 
+=== Reset Sandbox data
+In a sandbox environment it is a good pattern to reset the data periodically to the factory-defaults. To reset the data of the accelerator, the following steps are needed:
```
Comment (sw1ld) [inline arc42/src/07_deployment_view.adoc]: please change "accelerator" to "PSD2 Accelerator" or "sandbox"

File: arc42/src/07_deployment_view.adoc [hunk 2]
```diff
@@ -248,6 +248,13 @@ The CLI can echo the full schema instead of doing the actual migration, too. Thi
...
+
+ . Stop the running accelerator process by terminating the jar or by stopping the docker container.
+ . Delete the database data by redeploying your database instance. The procedure of resetting the database depends on your local deployment setup. Check the docs of your preferred database for further information.
+ . Restart the accelerator.
```
Comment (sw1ld) [inline arc42/src/07_deployment_view.adoc]: "docs" can be replaced with "documentation"

File: arc42/src/07_deployment_view.adoc (hunk 1 above)
Comment (wandi34) [reply arc42/src/07_deployment_view.adoc]: Done

File: arc42/src/07_deployment_view.adoc (hunk 2 above)
Comment (wandi34) [reply arc42/src/07_deployment_view.adoc]: Done

**Pull request merged**